core-library/src/main/java/org/silverpeas/core/index/indexing/parser/ps/PsParser.java

```diff
@@ -90,6 +87,7 @@ public void outPutChar(Writer out, BufferedReader buffer) throws IOException {
           } else {
             out.write(charr);
           }
+          break;
         case '\n':
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce this switch case number of lines from 6 to at most 5, for example by extracting code into methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1151)